windows/CodePushLifecycleEventListener.cs

```diff
@@ -0,0 +1,44 @@
+﻿using ReactNative.Bridge;
+using System;
+
+namespace CodePush.ReactNative
+{
+    internal class CodePushLifecycleEventListener : ILifecycleEventListener
+    {
+        private DateTime? _lastSuspendDate = null;
+        private Action _loadBundleAction;
+
```

Comment:
I thought you didn't want the word "resume" in it. How about "MinimumBackgroundListener"? Right now all this listener does is perform an action once the app has been in the background for x number of seconds.